imports/plugins/core/orders/client/templates/list/summary.html

```diff
@@ -63,5 +63,21 @@
       </div>
     </div>
 
+    {{#if hasAdminAccess}}
```

Comment:
Why an empty div at all? Or is there something that should be there that's not in there right now?